src/deviceinfoservice.cpp

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright (C) 2024 - AsteroidOS
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <QDBusInterface>
+#include <QDBusMessage>
+#include <QDebug>
+#include <QFile>
+
+#include <systemsettings/aboutsettings.h>
+
+#include "deviceinfoservice.h"
+#include "characteristic.h"
+#include "common.h"
+
+////////////////////////////////////////////////////////////////////////////////
+
+// #define INFO_MODEL_NO_UUID     "00002A24-0000-1000-8000-00805F9B34FB"
```

Comment:
These macro comments shouldn't be kept. People can refer to the definition of the actual macro.